Controller/EditLanguage.php

```diff
@@ -200,8 +209,8 @@ protected function editAction()
 
         $language = $this->getLanguageModel();
         $language->description = $this->request->request->get('description', '');
-        $language->idcontacto = ('' === $this->request->request->get('idcontacto', '')) ? null : $this->request->request->get('idcontacto', '');
-        $language->parentcode = ('' === $this->request->request->get('parentcode', '')) ? null : $this->request->request->get('parentcode', '');
+        $language->idcontacto = $this->request->request->get('idcontacto');
```

Comment:
$language->idcontacto = ('' === $this->request->request->get('idcontacto', '')) ? null : $this->request->request->get('idcontacto', '');

Is not equal to:

$language->idcontacto = $this->request->request->get('idcontacto');